src/arv-fake-camera.xml

```diff
@@ -222,6 +222,9 @@
 		<EnumEntry Name="Mono8" NameSpace="Standard">
 			<Value>17301505</Value>
 		</EnumEntry>
+		<EnumEntry Name="RGB8" NameSpace="Standard">
+			<Value>35127316</Value>
+		</EnumEntry>
```

Comment:
Actually, this edit slipped in too. I was trying to figure out how get different pixel formats, and ultimately realised what was needed was a custom genicam.xml file, which led me down to puzzling over the use of arv_set_fake_camera_genicam_filename.